bin/upload.js

```diff
@@ -9,6 +9,30 @@ import {argv} from 'yargs';
 
 import {NormandyApi} from '../lib/api';
 import {Action} from '../lib/models';
+import {localPath} from '../lib/utils';
+
+
+// Load config.json and figure out which environment we're using.
+let config = null;
+let config_env = argv.env || 'default';
+try {
+    config = JSON.parse(fs.readFileSync(localPath('config.json')));
+    if (!(config_env in config)) {
+        console.error(`No environment ${config_env} found in config.json.`);
+        process.exit();
+    } else {
+        config = config[config_env];
+    }
```

Comment:
I think I'd move checking for a named environment out of the try block, since it isn't going to throw an error you want to catch.